HW02_B_Mimic_starter.Rmd

```diff
@@ -36,6 +36,15 @@ Try for all 4, but if you are struggling don't worry about it. Try your best for
 ```{r, echo = T, include = TRUE}
 data("diamonds")
 #hint think about the *position* the bars are in...
+ggplot(diamonds, aes(cut, fill = clarity)) +
+  geom_bar(position = "dodge") + 
+  labs(x = "Diamond Cut", y = "Number of Diamonds", title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  theme(plot.title = element_text(hjust = 0.5)) +
+  annotate("rect", xmin = 4.5, xmax = 5.5, ymin = 0, ymax = 5000, alpha = .3) +

```

Comment:
I was not able to figure out how to add this element, I will make note of annotate().